snc.sh

```diff
@@ -177,12 +177,6 @@ ${YQ} write --inplace $INSTALL_DIR/manifests/cluster-ingress-02-config.yml spec[
 ${YQ} write --inplace $INSTALL_DIR/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value[domainMemory] 12288
 ${YQ} write --inplace $INSTALL_DIR/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value[domainVcpu] 6
 
-# Start the cluster with 10GB memory and 4 CPU create and wait till it finish
-# Todo: we need to remove this once stop building 4.2 bits.
-# For 4.3 this is ignored.
-export TF_VAR_libvirt_master_memory=10192
-export TF_VAR_libvirt_master_vcpu=4
-
```

Comment:
Maybe mention in the log that snc is already modifying `openshift/99_openshift-cluster-api_master-machines-0.yaml` so that it uses 12GB RAM and 6 CPUs

Reply:
But feel free to push this (`Remove env variable for master resources (mem/cpu) `) now even without the other patch which I haven't looked at yet :)